src/workloads/docs/HelloWorldFtdc.yml

```diff
@@ -0,0 +1,25 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/stm"
+
+Actors:
+- Name: HelloWorld
+  Type: HelloWorld
+  Threads: 2
+  Phases:
+  - Message: Hello Phase 0 🐳
+    Duration: 50 milliseconds
+    # GlobalRate: 99 per 88 nanoseconds
+    # SleepBefore: 11 milliseconds
+    # SleepAfter: 17 microseconds
+    # MetricsName: 🐳Message
+  - Message: Hello Phase 1 👬
+    Repeat: 100
+  - ExternalPhaseConfig:
+      Path: ../../phases/HelloWorld/ExamplePhase2.yml
+      Key: UseMe  # Only load the YAML structure from this top-level key.
+      Parameters:
+        Repeat: 2
+
+Metrics:
```

Comment:
Since the distinction between different metric formats is subtle enough, my vote is to not introduce the notion at all to new workload authors, who would be the target audience to helloworld workloads. Instead, suggest adding this section to the existing hello world workload in a commented-out format and add a comment above to say that people can use the ftdc metrics format instead of the default csv (I wouldn't say "Cedar" here since it could confuse the reader) format if they expect to record more than XYZ (10M?) data points.